pkgs/applications/graphics/photoflow/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, gettext, glib, libxml2, pkgconfig, swig, automake, gobjectIntrospection, cmake, ninja, libtiff, libjpeg, fftw, exiv2, lensfun, gtkmm2, libraw, lcms2, libexif, vips, expat, pcre, pugixml }:
+
+stdenv.mkDerivation {
+  name = "photoflow-0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "aferrero2707";
+    repo = "PhotoFlow";
+    rev = "914561fcb1ca97de4814c4024d9a76e4b435c2af";
+    sha256 = "06gsw0yhkdnhgcvnbsc5s4agm678ap10djvji8m2qssgnjn26vgs";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    glib
+    libxml2
+    pkgconfig
+    swig
+    automake
+    gobjectIntrospection
+    cmake
+    ninja
+  ];
+
+  buildInputs = [
+    libtiff
+    libjpeg
+    fftw
+    exiv2
+    lensfun
```

Comment:
For some reason, photoflow builds its own version of lensfun. Could you fix it?

Reply:
This should do it.
```
  cmakeFlags = [
    "-DBUNDLED_EXIV2=OFF"
    "-DBUNDLED_LENSFUN=OFF"
    "-DBUNDLED_GEXIV2=OFF"
  ];
```

Reply:
Done